process/smartContract/process.go

```diff
@@ -593,12 +641,9 @@ func (sc *scProcessor) processSCPayment(tx data.TransactionHandler, acntSnd stat
 		return nil
 	}
 
-	err := acntSnd.SetNonceWithJournal(acntSnd.GetNonce() + 1)
-	if err != nil {
-		return err
-	}
+	acntSnd.SetNonce(acntSnd.GetNonce() + 1)
```

Comment:
I would have an increaseNonce function only.

Reply:
I need SetNonce() func because of the acc.SetNonce(outAcc.Nonce) from processSCOutputAccounts().

Reply:
Done.